docs/api/ethereum.md

```diff
@@ -1009,6 +1009,32 @@ const accountId = '0xcc567f78e8821fb8d19f7e6240f44553ce3dbfce';
 const abilities = await ledger.getAbilities(accountId);
 ```
 
+### getAccountAssetIdAt(accountId, index)
```

Comment:
Add a warning that some existing contracts may not support this (assets deployed trough framework do). Basically if the return is null either contract does not support the function or index does not exist. 

Add this for both functions.